shell/config/router/routes.js

```diff
@@ -494,17 +494,15 @@ export default [
         path:      '/c/:cluster/:product/:resource/:id',
         component: () => interopDefault(import('@shell/pages/c/_cluster/_product/_resource/_id.vue')),
         name:      'c-cluster-product-resource-id',
-        meta:      { asyncSetup: true }
       }, {
         path:      `/c/:cluster/:product/${ VIRTUAL_TYPES.PROJECT_SECRETS }/:namespace/:id`,
         component: () => interopDefault(import(`@shell/pages/c/_cluster/explorer/${ VIRTUAL_TYPES.PROJECT_SECRETS }.vue`)),
         name:      `c-cluster-product-${ VIRTUAL_TYPES.PROJECT_SECRETS }-namespace-id`,
-        meta:      { asyncSetup: true }
+        meta:      { resource: 'projectsecret' },
```

Comment:
This could be achieved different ways, it's really so we know what resource is being accessed in ResourceDetail/index. I'd definitely be open to other options but we do have.